aesara/tensor/basic.py

```diff
@@ -3634,6 +3635,47 @@ def swapaxes(y, axis1, axis2):
     return y.dimshuffle(li)
 
 
+def moveaxis(a, source, destination):
+    """Move axes of a TensorVariable to new positions.
+
+    Other axes remain in their original order.
+
+    Parameters
+    ----------
+    a : TensorVariable
+        The TensorVariable whose axes should be reordered.
+    source : int or sequence of int
+        Original positions of the axes to move. These must be unique.
+    destination : int or sequence of int
+        Destination positions for each of the original axes. These must also be
+        unique.
+
+    Returns
+    -------
+    result : TensorVariable
+        TensorVariable with moved axes.
+
+    """
```

Comment:
We're using type hints now instead of docstring type descriptions.

Reply:
Is it fine to still include them in the docstrings?

Reply:
I'd rather not, just in case it messes with the way docstrings are rendered in Sphinx; especially, when we start using the type hints in the documentation.

Reply:
Can you point me to an example of docstrings with the format we want to follow? Should I just remove whatever was after `:` in the parameter enumeration?

Reply:
It's best to just remove the type info from the docstring.  With the type hints in place, it will just amount to extra work later on when/if those types are adjusted.